source/postprocess/point_values.cc

```diff
@@ -166,6 +178,11 @@ namespace aspect
                             "The list of points at which the solution should be evaluated. "
                             "Points need to be separated by semicolons, and coordinates of "
                             "each point need to be separated by commas.");
+          prm.declare_entry("Use natural coordinates", "false",
+                            Patterns::Bool (),
+                            "Whether or not the Evaluation points are specified in "
+                            "the natural coordinates of the geometry model, e.g. "
+                            "spherical coordinates for the spherical shell.");
```

Comment:
Maybe add specifically the geometry models which are supported and are not supported at the moment.